core/modules/user/user.admin.inc

```diff
@@ -168,6 +168,45 @@ function user_admin_settings($form, &$form_state) {
     }
   }
 
+  // Password settings.
+  $form['password_policy'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Password policy'),
+    '#weight' => -1,
+  );
+  $output = '';
+  $output .= '<p>' . t('The default criteria for a strong password include checking that it is:') . '</p>';
+  $output .= '<ul><li>' . t('not the same as the username') . '</li>';
+  $output .= '<li>' . t('not the same as the e-mail address') . '</li>';
+  $output .= '<li>' . t('sufficiently complex: containing a number of lower- and uppercase letters, digits and symbols') . '</li>';
+  $output .= '<li>' . t('at least as many characters as the minimum number set below') . '</li></ul>';
+  $form['password_policy']['user_password_criteria'] = array(
```

Comment:
I have been advised that it is best to output things that are to be displayed to users through `theme()` functions where possible. Also, we should be rendering that as a help text, so how about this?:

```suggestion
  $output = '';
  $criteria = array();
  $criteria[] = t('not the same as the username');
  $criteria[] = t('not the same as the e-mail address');
  $criteria[] = t('sufficiently complex: containing a number of lower- and uppercase letters, digits and symbols');
  $criteria[] = t('at least as many characters as the minimum number set below');

  $output .= '<p>' . t('The default criteria for a strong password include checking that it is:') . '</p>';
  $output .= theme('item_list', array('items' => $criteria));
  $form['password_policy']['user_password_criteria'] = array(
    '#type' => 'help',
```